dnsext-full-resolver/dug/dug.hs

```diff
@@ -105,16 +105,16 @@ defaultOptions =
 
 main :: IO ()
 main = do
+    runInitIO $ do
```

Comment:
Please add a comment why this logic should be here.

Reply:
https://github.com/kazu-yamamoto/dnsext/pull/103/commits/413d784bb589475edea01c665235d35784e2e8de